lib/main.dart

```diff
@@ -18,10 +29,13 @@ class CalmQuestApp extends StatelessWidget {
       initialRoute: '/',
       routes: {
         '/': (context) => const HomeScreen(),
-        '/todays_activities': (context)=> const TodaysActivitiesScreen(),
+
+        '/todays_activities': (context) => const TodaysActivitiesScreen(),
         '/achievements': (context) => const AchievementsScreen(),
         '/settings': (context) => const SettingsScreen(),
+        '/mood_journal': (context) => MoodSelectionScreen(storage: storage), // Pass Storage instance
```

Comment:
Activities do not go on the nav bar or in the routes. Check how the other activities are implemented